catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java

```diff
@@ -97,7 +97,7 @@
 @Command(
   scope = CatalogCommands.NAMESPACE,
   name = "export",
-  description = "Exports Metacards and history from the current Catalog"
+  description = "Exports Metacards, history, and content(products/resources) from the current Catalog"
```

Comment:
Should we just say `content`? If it's a filesystem/CDM resource it won't be exported, will it?

Reply:
Currently it looks like we only export things with `content:` in the URI, though i'd imagine we could look into changing that since it should support any resource you can get through the catalog framework.

![image](https://user-images.githubusercontent.com/9013416/64037376-48066180-cb0a-11e9-9b43-c3f25dda03c2.png)

Reply:
Updated to match nomenclature of catalog framework -- `resource`